packages/@coorpacademy-components/src/atom/chip/index.js

```diff
@@ -0,0 +1,77 @@
+import React, {useMemo, useState, useCallback} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import get from 'lodash/fp/get';
+import Color from 'colorjs.io';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import Provider from '../provider';
+import style from './style.css';
+
+const DELTA_LUMINOSITY = 0.08;
+const DEFAULT_BACKGROUND_COLOR = '#0061FF';
```

Comment:
color value #0061FF could be added to https://github.com/CoorpAcademy/components/blob/master/packages/@coorpacademy-components/src/variables/colors.ts#L9 as coorp manager blue (`cm_primary_blue`) 
(and type declaration simplified with an `as const`)